cron/go.mod

```diff
@@ -1,5 +0,0 @@
-module github.com/ossf/scorecard/cron
```

Comment:
This should not be deleted.

Reply:
Any reason to have this as a separate module? I think the convention is to have a single module per repo unless absolutely required - https://github.com/golang/go/wiki/Modules#should-i-have-multiple-modules-in-a-single-repository

Reply:
I am not against it, as long the dependencies are in main modules `go.mod` which I see you have recently updated it https://github.com/ossf/scorecard/pull/344/commits/b58a0f38095c1138eed267feb43f5b61c93c9b95

Reply:
Thanks